plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -349,6 +361,23 @@ class NewServiceFormModal extends Component {
     ];
   }
 
+  /**
+   * Normalize the given `service` instance/config in order to match the
+   * specifications required by the editing/reviewing components
+   *
+   * @param {Pod|Application|Object} service - The service to normalize
```

Comment:
Please adjust the documentation... Input is `Service` and output is `ServiceSpec`.